Add get volume function

Liborsaf opened the pull request.

Liborsaf committed:
Add get volume function

alexmohr commented:
Thanks for the contribution. I removed travis and added github ci instead. In turn I had to fix some new linter warnings, so you have to fix some conflicts now. Sorry :/
Also please add a unit test for the new method

Liborsaf commented:
> Thanks for the contribution. I removed travis and added github ci instead. In turn I had to fix some new linter warnings, so you have to fix some conflicts now. Sorry :/ Also please add a unit test for the new method

No problem, will look into that in few minutes :)

Liborsaf committed:
Fix linting problems

Liborsaf committed:
Add unit tests for get volume function

Liborsaf commented:
Not very experienced in doing unit tests, so hopefully this is good :D

alexmohr commented on sonyapilib/device.py:
```diff
@@ -359,12 +360,25 @@ def _parse_dmr(self, data):
-                self.av_transport_url = f"{lirc_url.scheme}://{lirc_url.netloc.split(':')[0]}"\
-                    f":{self.dmr_port}{transport_location}"
+                service_id_text = service_id.text
+                # print(service_id_text)
```
remove comment (all commented out print statements you added )

Liborsaf replied:
Done 👍🏻

alexmohr reviewed (changes requested):
Thanks, the test is good just one more minor finding and then this is good to go :)

Liborsaf committed:
Remove all commented statements
